src/Components/profile/Profile.js

```diff
@@ -36,10 +36,18 @@ const Profile = () => {
   const [pageCount, setPageCount] = useState(0);
   const [eventsListUser, setEventListUser] = useState([]);
 
+  useEffect(() => {
+    setTextAvatar(`${user.name[0]} ${user.lastName[0]}`);
+    setUserName(`${user.name} ${user.lastName}`);
+    setNameUser(user.name);
+    setLastNameUser(user.lastName);
+    setEmailUser(user.email);
```

Comment:
we dont need an useEffect, we dont need state variables neither, lets do this instead:
```
const textAvatar = ${user.name[0]} ${user.lastName[0]}
const userName = ${user.name} ${user.lastName}
const name = user.name
const lastName = user.lastName
const email = user.email
```